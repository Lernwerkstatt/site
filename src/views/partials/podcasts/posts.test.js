const modal = require("./posts.hbs");

describe("posts.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      modal({
        title: "Post",
        date: "01.01.2000",
        author: "unknown",
        video_id: "xyz",
      })
    ).toMatchSnapshot();
  });
});
