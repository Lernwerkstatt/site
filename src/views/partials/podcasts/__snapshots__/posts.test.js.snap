// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`posts.hbs should match the snapshot 1`] = `
"<div class="col-xs-8 col-md-6 col-lg-4">
  <div class="card mt-3 mb-5" style="height: 22rem; max-width: 100%">
    <img class="card-img-top" loading="lazy" style="height: 14rem; object-fit: cover;" src="https://img.youtube.com/vi/xyz/0.jpg"
      alt="Post">
    <div class="card-body">
      <h2 class="card-title" style="overflow-wrap: normal"><a href="/podcasts/">Post</a></h2>
      <p class="card-text"><small>unknown, 01.01.2000</small></p>
    </div>
  </div>
</div>
"
`;
